mkdocs.yml

```diff
@@ -189,6 +189,18 @@ nav:
               - '    Events': Tutorials/Getting-Started/Basics/Events.md
               - 'Objectives': Tutorials/Getting-Started/Basics/Objectives.md
               - 'Conditions': Tutorials/Getting-Started/Basics/Conditions.md
+      - 'Advanced Tutorials':
+          - 'About': Tutorials/Advanced-Tutorials/About.md
+          - 'Daily Quests':
+              - 'Overview': Tutorials/Advanced-Tutorials/Daily-Quests/Overview.md
+              - '   Pickrandom': Tutorials/Advanced-Tutorials/Daily-Quests/Pickrandom.md
```

Comment:
The icon on my webbrowser looks bad... maybe we need to find something else here